ddtrace/internal/writer/writer.py

```diff
@@ -161,6 +161,8 @@ def __init__(
         report_metrics=True,  # type: bool
     ):
         # type: (...) -> None
+        super(HTTPWriter, self).__init__()
+        TraceWriter.__init__(self)
```

Comment:
🤔  we shouldn't be needing to call all the `__init__`s manually if every other `__init__` in the mro calls `super` I think